workshops/interactive_maps/README.md

```diff
@@ -0,0 +1,279 @@
+---
+name: 'Interactive Maps'
+description: 'Learn JS and HTML while making a fun interactive map'
+author: '@LonaDotExe'
+---
+
+# Making an interactive map using Javascript 
+By the end of this tutorial, you will learn to use functions and IF-ELSE statements in nothing but Vanilla JS to create some really good-looking interactive maps. This workshop will also teach you the basics of HTML and CSS. As a bonus, read till the end for a fun programming fact.
+
+- Demo - https://interactive-maps.jeswinsunsi.repl.co/
+- Final code - https://repl.it/@JeswinSunsi/Interactive-Maps#index.html
+
+![gif demo](https://cloud-fmga8dmlt.vercel.app/gif_one.gif)
+### 1 - Prerequisites
+- Basic understanding of functions 
+- Amateur knowledge of HTML and CSS
+
+In case you don't know either of those, don't worry. We'll go through each line of code in depth.
+
+### 2 - Tools and Resources
+- For this tutorial, we'll be using repl.it, an online browser-based IDE
+- For editing the photos, we can use Pixlr.com or any other full fledged photo editor
+- I recommend going through https://www.w3schools.com/js/js_functions.asp for further clarification with the JS part
+
+
+### 3 - Designing the image 
+Any suitable image will work, but I have chosen an animated isometric map. The image was of the JPG format, but I removed the background to get a png image. Then from that one image, I selected specific areas and created 5 different images, each with a different area highlighted in red.
+For a visual understanding, here is the original image followed by an edited one.
+
+![image 1](https://cloud-fyde5ir0a.vercel.app/image.png)
+![image 2](https://cloud-jmd1dy3k8.vercel.app/image.png)
+
+### 4 - Coding it up
+##### The boilerplate
+Head over to https://repl.it. Creating an account is of personal preference, be warned that you may lose your work if you don't have an account. Click on start coding and search for HTML, CSS, JS. Once you create a repl, you'll end up at this page.
```

Comment:
you can tell the people to use this link https://repl.it/languages/html
instead of telling them to search for it, It will be easy for them that way!